### PR TITLE
[libosmium ] added new port

### PR DESCRIPTION
**Pull request by am2222**
**Describe the pull request**

- What does your PR fix? Fixes #
Adds libosmium port https://osmcode.org/libosmium/
- Which triplets are supported/not supported? Have you updated the CI baseline?
CI baseline is not edited
- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes
- Original contributor: @jremillard
https://github.com/microsoft/vcpkg/issues/7526#issuecomment-640978914

### CODE REVIEW

**Comment by jremillard** [comment]
I think the protozero is needed first. https://github.com/jremillard/vcpkg/commit/db94c318917816bed6ad88fa665c2c6e497762c2

**Comment by am2222** [comment]
> I think the protozero is needed first. [jremillard@db94c31](https://github.com/jremillard/vcpkg/commit/db94c318917816bed6ad88fa665c2c6e497762c2)


But that package has already added to the vcpkg. Here is its PR
https://github.com/microsoft/vcpkg/pull/11652
But do we actually need this package for mapnik? on my machine it builds with no issues.

File: ports/libosmium/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA # Disable this option if project cannot be built with Ninja
+    OPTIONS -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF
+)
+
+vcpkg_install_cmake()
+
+# Handle copyright
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/libosmium RENAME copyright)
```
Comment (LilyWangL) [inline ports/libosmium/portfile.cmake:28]: Can you please use command `file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)` to handle copyright?

File: ports/libosmium/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,30 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO osmcode/libosmium
+    REF v2.15.5
+    SHA512 a4972901db8ed89302e6ba15fd104543b5e36a41bc83daf8f6f6fb29ce73b0dbd8596de801d099a33df413b26eec1b3a6f4f0d669936ecc6d25f88d783468a59
+)
+#     [PATCHES <patch1.patch> <patch2.patch>...]
```
Comment (LilyWangL) [inline ports/libosmium/portfile.cmake:7]: Please remove the following useless comment lines.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

**Comment by jremillard** [comment]
I can't see the logs for the automated checks? 
@am2222, this package isn't needed for mapnik.

**Comment by LilyWangL** [comment]
> I can't see the logs for the automated checks?
> @am2222, this package isn't needed for mapnik.

The CI `x86-windows` build log download URL: https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/38542/artifacts?artifactName=x64-windows-static%20port%20build%20failure%20logs&api-version=5.1&%24format=zip

**Review by am2222 — COMMENTED** [review]
changes are applied

**Review by LilyWangL — APPROVED** [review]

**Review by BillyONeal — APPROVED** [review]

**Pull request merged**